### DISCUSSION TITLE
module 'monai.data' has no attribute 'NibabelWriter'

### DISCUSSION THREAD

**Discussion by JunTan2022** [Project-MONAI/MONAI#6373] [Q&A]
Hi,

I have just installed MONAI version 1.1.0 on my conda environment.

When I run `writer = monai.data.NibabelWriter()`, I got the following error:

AttributeError: module 'monai.data' has no attribute 'NibabelWriter'

Do you have any ideas about what might be going on? I am grateful for your help-- thanks in advance!

**Comment by JunTan2022** [comment]
I uninstall, reinstall MONAI and reselect my conda kernel as the interpreter and everything works, though I still don't why that error occurred.

**Reply by KumoLiu** [reply]
Hi @JunTan2022, there maybe some version conflict. Anyway, glad to see it can work now.
